Cargo.toml

```diff
@@ -13,12 +13,13 @@ categories = ["text-processing"]
 
 [dependencies]
 log = "0.4"
-memmap2 = { version = "0.3", optional = true }
 ttf-parser = "0.12.1"
+memmap2 = { version = "0.3", optional = true }
 
 [dev-dependencies]
 env_logger = { version = "0.8", default-features = false }
 
 [features]
 default = ["fs"]
-fs = ["memmap2"] # allows local filesystem interactions
+fs = [] # allows local filesystem interactions
+memmap-fonts = ["memmap2"]
```

Comment:
`memmap` should be enough. No need for longer names. And add a comment like this:

```
memmap = ["memmap2"] # allows font files memory mapping, greatly improves performance 
```

Reply:
Also, I think `memmap` should depend on `fs` too, so `["fs", "memmap2"]`